panther_manager/src/docking_manager_node.cpp

```diff
@@ -0,0 +1,125 @@
+// Copyright 2024 Husarion sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "panther_manager/docking_manager_node.hpp"
+
+#include <any>
+#include <chrono>
+#include <functional>
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "ament_index_cpp/get_package_share_directory.hpp"
+#include "behaviortree_ros2/ros_node_params.hpp"
+#include "rclcpp/rclcpp.hpp"
+
+#include "panther_utils/moving_average.hpp"
+
+#include <panther_manager/behavior_tree_manager.hpp>
+#include <panther_manager/behavior_tree_utils.hpp>
```

Comment:
According to code guidelines.
```suggestion
#include <ament_index_cpp/get_package_share_directory.hpp>
#include <behaviortree_ros2/ros_node_params.hpp>
#include <rclcpp/rclcpp.hpp>

#include <panther_manager/behavior_tree_manager.hpp>
#include <panther_manager/behavior_tree_utils.hpp>
#include <panther_utils/moving_average.hpp>
```